swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
```

Comment:
[nit] in general, I prefer to space around `=`

```suggestion
    def __init__(self, inventory = None):
```